src/runtime/composables/useSlideover.ts

```diff
@@ -1,63 +1,84 @@
-import { ref, inject } from 'vue'
+import { inject } from 'vue'
 import type { ShallowRef, Component, InjectionKey } from 'vue'
 import { createSharedComposable } from '@vueuse/core'
 import type { ComponentProps } from '../types/component'
-import type { Slideover, SlideoverState } from '../types/slideover'
+import type { Slideover, SlideoverInstance } from '../types/slideover'
 
-export const slidOverInjectionKey: InjectionKey<ShallowRef<SlideoverState>> = Symbol('nuxt-ui.slideover')
+export const slidOverInjectionKey: InjectionKey<ShallowRef<SlideoverInstance[]>> = Symbol('nuxt-ui.slideover')
 
 function _useSlideover () {
-  const slideoverState = inject(slidOverInjectionKey)
-
-  const isOpen = ref(false)
+  const slideoverInstances = inject(slidOverInjectionKey)
 
   function open<T extends Component> (component: T, props?: Slideover & ComponentProps<T>) {
-    if (!slideoverState) {
+    if (!slideoverInstances) {
       throw new Error('useSlideover() is called without provider')
     }
+    
+    const instance = createInstance(component, props)
 
-    slideoverState.value = {
-      component,
-      props: props ?? {}
-    }
+    slideoverInstances.value = [
+      ...slideoverInstances.value,
+      instance
+    ]
 
-    isOpen.value = true
+    return instance
   }
 
-  async function close () {
-    if (!slideoverState) return
+  async function close (id: number) {
+    const slideoverInstance = slideoverInstances.value.find((slideover) => slideover.id === id)
+    
+    if (!slideoverInstance) return
 
-    isOpen.value = false
+    slideoverInstance.isOpen = false
+    
+    slideoverInstances.value = [
+      ...slideoverInstances.value.filter((slideover) => slideover.id !== id),
+      {
+        ...slideoverInstance,        
+        isOpen: false      
+      }
```

Comment:
You already update `isOpen` a few lines above
```suggestion
      slideoverInstance
```